policyengine_uk/variables/gov/dwp/pension_credit/pension_credit_earnings.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_uk.model_api import *
+
+
+class pension_credit_earnings(Variable):
+    label = "Earnings for Pension Credit"
+    entity = BenUnit
+    definition_period = YEAR
+    value_type = float
+    unit = GBP
+    reference = "https://www.legislation.gov.uk/uksi/2002/1792/regulation/17A"
+
+    def formula(benunit, period, parameters):
```

Comment:
You can just do `adds = "gov.dwp.pension_credit.income.earnings_sources"`

US example [here](https://github.com/PolicyEngine/policyengine-us/blob/178781ed10dd096c1edae26f78d0130fa9b430b4/policyengine_us/variables/gov/states/ia/tax/income/net_income/ia_gross_income.py#L14)